pb/cmd_pb/cmd.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package ziti.cmd.pb;
+option go_package = "github.com/openziti/fabric/pb/cmd_pb";
+
+enum CommandType {
+  Zero = 0;
+  CreateEntityType = 1;
+  UpdateEntityType = 2;
+  DeleteEntityType = 3;
+}
+
+message CreateEntityCommand {
+  string entityType = 1;
+  bytes entityData = 2;
```

Comment:
Does this need to be versioned? Or is versioning handled inside of `entityType`?

What happens if we have two different controller versions that expect two different byte interpretations of `entityData`?

Reply:
I think we can handle versioning either in the message header (outside of the prototype) or in the entityData. We can also add a version field later in the protobuf without breaking backwards compatibility. Versioning still requires more thought.